www/Model/User.class.php

```diff
@@ -160,9 +160,9 @@ public function formatList(): array
     /**
      * @param string $isVerified
      */
-    public function setIsVerified(string $isVerified): void
+    public function setIsVerified(): void
     {
-        $this->isVerified = $isVerified;
+        $this->isVerified = ($this->isVerified == 1 ? 0 : 1);
```

Comment:
Là on inverse le status de IsVerified non ?
Si c'est ce que tu veux faire tu peux directement mettre !$this->isVerified

Reply:
non parce que en db c'est 0 ou 1 et en php 0 et 1 c'est pas strictement égale à true et false

Reply:
Hmmm ok ok je vois, mais du coup la fonction elle change la valeur actuelle, ça change un peu de comment elle est utilisée du coup

Reply:
en vrai dès que tu as récup l'utilisateur en db la seul raison de le modifier c'est de l'inverser donc autant faire comme ca

Reply:
Oui, mais, le nom de la fonction ne correspond plus à ce qu'elle fait